League Sandbox Auto Setup/Form1.cs

```diff
@@ -340,5 +373,16 @@ private void startVisualStudioFirstRun()
                 }
             });
         }
+
+        private void BrowseButton_Click(object sender, EventArgs e)
+        {
+            using (var SelectPath = new FolderBrowserDialog())
```

Comment:
Should be `selectPath` as it is a variable.

Reply:
Updated, thanks.